lib/Tool/Text.php

```diff
@@ -77,6 +79,12 @@ public static function wysiwygText(?string $text, array $params = []): ?string
                                     $path .= '#' . $urlParts['fragment'];
                                 }
                             }
+
+                            $site = Frontend::getSiteForDocument($element);
+                            if ($site instanceof Site) {
+                                $path = Tool::getRequestScheme() . '://' . $site->getMainDomain() . preg_replace('~^' . $site->getRootPath().'~','', $path);
```

Comment:
Should we escape it?
```suggestion
                                $path = Tool::getRequestScheme() . '://' . $site->getMainDomain() . preg_replace('~^' . preg_quote($site->getRootPath(), '~') . '~', '', $path);
```

Reply:
mhh that's a good point